questdb/run.sh

```diff
@@ -13,7 +13,7 @@ cat queries.sql | while read query; do
     echo "$query";
     for i in $(seq 1 $TRIES); do
         curl -sS --max-time 600 -G --data-urlencode "query=${query}" 'http://localhost:9000/exec?timings=true' 2>&1 | grep '"timings"' ||
-            (questdb-7.3-rt-linux-amd64/bin/questdb.sh stop >/dev/null 2>&1; questdb-7.3-rt-linux-amd64/bin/questdb.sh start >/dev/null 2>&1; sleep 5; echo 'null')
+            (questdb/bin/questdb.sh stop >/dev/null 2>&1; questdb/bin/questdb.sh start >/dev/null 2>&1; sleep 5; echo 'null')
```

Comment:
Initial version of these scripts was contributed by Alexey Milovidov and these restarts were already there. I was under the impression that all databases are put in the same conditions and there are restarts in other scripts too (didn't check them). Isn't that the case?

Reply:
> there are restarts in other scripts too (didn't check them). Isn't that the case?

no, that's why it caught my eye. prob he had to do it, because many queries ran with some errors.

[compatible/questdb/result/c6a.4xlarge.txt](https://github.com/ClickHouse/ClickBench/commit/bdd944b584f030731814b6c67156ed798525a59f#diff-12995188669d484ec2bf2ce1ecc42aac29bd678f4da04412b1dfe3995f00a01f)

if it is not the case now, I think it is better to remove restarts

Reply:
Oh, good to know the reason.

> of course we don't have to do it in this pr.

We'll get rid of these restarts in the next ClickBench result update.